distribution/pull_v2.go

```diff
@@ -487,6 +486,14 @@ func (p *v2Puller) pullV2Tag(ctx context.Context, ref reference.Named, platform
 }
 
 func (p *v2Puller) pullSchema1(ctx context.Context, ref reference.Reference, unverifiedManifest *schema1.SignedManifest, platform *specs.Platform) (id digest.Digest, manifestDigest digest.Digest, err error) {
+	if platform != nil {
+		// Early bath if the requested OS doesn't match that of the configuration.
```

Comment:
`Early bath`?

Reply:
Comment was already there. I can only assume John wrote that, as it looks to be an 🇬🇧 expression https://dictionary.cambridge.org/dictionary/english/an-early-bath